hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -155,18 +157,60 @@ private StoreFileInfo(final Configuration conf, final FileSystem fs, final FileS
     } else if (isHFile(p) || isMobFile(p) || isMobRefFile(p)) {
       // HFile
       if (fileStatus != null) {
-        this.createdTimestamp = fileStatus.getModificationTime();
-        this.size = fileStatus.getLen();
-      } else {
-        FileStatus fStatus = fs.getFileStatus(initialPath);
-        this.createdTimestamp = fStatus.getModificationTime();
-        this.size = fStatus.getLen();
+        this.localStatus = fileStatus;
+      }else{
```

Comment:
Please keep the style of the surrounding file; i.e. space between '}' and 'else' and after.